tests/std/tests/P0896R4_ranges_alg_generate_n/test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <range_algorithm_support.hpp>
+#include <ranges>
+#include <utility>
+
+constexpr void smoke_test() {
+    using ranges::generate_n, ranges::iterator_t;
+    using std::same_as;
+
+    int output1[] = {13, 42, 1367};

```

Comment:
Should this be `output` as there is only one?

Reply:
Ditto "should we unbrace the codeblock"?